modules/serverless-create-func-kn.adoc

```diff
@@ -32,10 +28,10 @@ envs: {}
 +
 [source,terminal]
 ----
-$ kn func create <path> -r <registry> -l <runtime> -t <trigger> -i <image> -n <namespace>
+$ kn func create <path> -r <repository> -l <runtime> -t <trigger> -i <image> -n <namespace>
```

Comment:
It's OK to leave this as-is because `repository` is a valid option for `create` but it is not required, and it involves what we are calling in the upstream, "Language Packs" which have not yet been introduced in the OpenShift documentation. For this reason, I suggest the `-r <repository>` option for this example be omitted entirely. 

Also, a change related to the Language Pack push is that the `-t` flag indicates a `template` instead of a `trigger`.

Also - as `-n <namespace>` is not required and also unexplained in this document, I suggest it be removed as well.

And finally, `-i <image>` is no longer a valid option for create.

My apologies for missing these in the downstream Jira issue. All but the registry/repository changes happened in releases further back, and I  was only referencing changes from last to this in the issue. I didn't realize that this document was quite so out of date.

```suggestion
$ kn func create <path>  -l <runtime> -t <template>
```